### ISSUE TITLE
Fans stop working after using desktop mode for a while - occurs randomly

### ISSUE THREAD

**Issue by Maar10Herr** [ValveSoftware/SteamOS#1531]
#### Your system information

* Steam client version: 1716584667
* SteamOS version: 3.5.19
* Opted into Steam client beta?: [No] 
* Opted into SteamOS beta?: [No] 
* Have you checked for updates in Settings > System?: [Yes]
* SteamID: 76561198350625099

#### Please describe your issue in as much detail as possible:
Since February my steam deck's fans randomly stop working and stay at 0rpm even if the system temperature is above 90c.
The issue occurs seemingly random and can sometimes be fixed by just putting the deck to sleep, but occasionally nothing works and I have to wait until the battery is fully drained.
The issue has usually occurs after desktop mode has been used but happens very sporadically.
Re-flashing SteamOS did not solve the issue.
The fan connector is also correctly seated and entering the bios the fans work 100% of the time.
#### Steps for reproducing this issue:
No clear way of reproducing but after re-flashing the deck today it first occurred while playing a game via yuzu added to steam as a non-steam game.
1. Yuzu was installed on the main drive via the latest app-image
2. No power profiles, system files, or anything was changed. Permissions were not modified.
3. Exiting desktop mode will cause this issue to occur about 5% of the time

My suspicion is that it's related to the desktop mode not properly functioning and some processes still running when in game mode.
The issue never occurs when the steam deck is charged from 0% battery or has been updated and has not been launched into desktop mode yet.
The issue is however not unique to non-steam games as I first noticed it occurring in Hades (bought and installed via steam).

Let me know which logs to share and I will do so if I get it to occur again. 
This is my first issue on GitHub so let me know if there's a problem. Just thought I'd ask here before RMA'ing.

**Comment by Maar10Herr** [comment]
> Really appreciate the reply thanks! I'm in NZ and there's no onshore valve support here so doing it tough through online support as well. Will perserve and see how we go - thanks again!

Thank you for reminding me! I have about 2 months left on my RMA period (it's 2 years for factory defects in the EU) so I better use my deck again with the newest software and document the issue.
The issue seems documented online but in all of the cases I have seen, they received a replacement device. I don't think that's easy to get outside the US so we might have to contact consumer protection agencies.

**Comment by Maar10Herr** [comment]
> so I better use my deck again with the newest software and document the issue

I do have 3 separate conversations with support over the years but have only been able to send it in once. But of course I got the device sent back without further information.

**Comment by thepaullarsen-svg** [comment]
It's only happening every couple of days at the moment so more annoying than deal breaking but definitely a concern. Did you ever do a factory reset?

**Comment by Maar10Herr** [comment]
I've actually done multiple resets, graphic driver resets, bios resets, software reflashes, you name it...
What you could try doing when it occurs is going into the bios or restarting the deck. For me, before SteamOS itself loads (so on the boot screen) the fans run perfectly, but as soon as I am in SteamOS they stop spinning. So it has to be a software issue.

**Comment by thepaullarsen-svg** [comment]
Yeah, its definitely not hardware. the fan spools up and down as it should 99% of the time - It just decides to stop completely under a certain type of load while in-game.

**Comment by thepaullarsen-svg** [comment]
It actually just happened now and I got a photo mid-game of the overlay reading 0 rpm. usually above 55c yes, and usually just putting it into sleep and then waking it up restarts the fan. sounds like your issue more serious.
